uv.lock

```diff
@@ -1972,6 +1974,19 @@ wheels = [
     { url = "https://files.pythonhosted.org/packages/a7/4b/8b78d126e275efa2379b1c2e09dc52cf70df16fc3b90613ef82531499d73/pytest_cov-4.1.0-py3-none-any.whl", hash = "sha256:6ba70b9e97e69fcc3fb45bfeab2d0a138fb65c4d0d6a41ef33983ad114be8c3a", size = 21949, upload-time = "2023-05-24T18:44:54.079Z" },
 ]
 
+[[package]]
+name = "pytest-rerunfailures"
```

Comment:
Ideally, we don't want that package to be included - we don't want flaky tests if possible.